### PR TITLE
Create Routine_Data_Manipulation.java

### PR DESCRIPTION
**Pull request by luckey112**
Hello friend
I am cretae a Routine Data Manipulation Program in Java Language.
Please kindly merge my PRs & Add level "hacktoberfest-accepted"
THANKU
Happy Hacking!

### CODE REVIEW

**Comment by luckey112** [comment]
I am waiting your good response

File: src/main/java/org/example/algorithms/Routine_Data_Manipulation.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/*
+Routine Data Manipulation Algorithm
+
+Start Code:
```
Comment (BimalRajGyawali) [inline src/main/java/org/example/algorithms/Routine_Data_Manipulation.java:4]: Package info is missing

**Review by BimalRajGyawali — CHANGES_REQUESTED** [review]
Remove underscores from file name

**Comment by BimalRajGyawali** [comment]
Thanks @luckey112 . I have requested some changes.

**Pull request closed without merging**